scipy/signal/wigner.py

```diff
@@ -0,0 +1,95 @@
+"""Tools based on the Wigner quasiprobability distributions.
+"""
+
+import numpy as np
+from scipy.fft import fft
+
+
+__all__ = ['wvd']
+
+
+def wvd(x, fs=1.0, resolution=1, win_size=None):
```

Comment:
Could you think of a more readable name? This is too Matlab-y. Maybe just `wigner_ville`?

Reply:
The less Matlab-y the better, especially because thier `wvd` is a mix of different methods related to the Wigner-Ville distribution. `wigner_ville` is fine with me.